scalameta/dialects/src/main/scala/scala/meta/dialects/package.scala

```diff
@@ -173,6 +181,7 @@ package object dialects {
     def allowOrTypes = Scala211.allowOrTypes
     def allowTraitParameters = Scala211.allowTraitParameters
     def allowLiteralTypes: Boolean = false // Scheduled to be included in 2.12.2
+    def allowTrailingComma: Boolean = true // Schedule to be included in 2.12.2
```

Comment:
Let's make both false and then update both to true when 2.12 is released (see https://github.com/scalameta/scalameta/issues/645).